prow/scripts/lib/gardener/aws.sh

```diff
@@ -115,6 +115,16 @@ gardener::test_fast_integration_kyma() {
     return
 }
 
+gardener::pre_upgrade_test_fast_integration_kyma() {
+    # TODO
```

Comment:
do you expect any difference for aws & gcp clusters? i would just the same command to trigger tests as in `azure.sh`

Reply:
Not really. I defined all these functions for all providers.